README.md

```diff
@@ -1,49 +1,51 @@
+# Goal
 
+The purpose of this repository is to provide a template for dockerizing an existing [shiny application](https://shiny.posit.co/), which handles an input folder containing (non-obligatory) data and an output folder where any results and log data exported from the app will be placed. 
+The shiny app should include an action button to "save and exit", which cleanly closes the app and stops the container.
 
 
-a simple shiny app in a docker container
+# Overview of the setup
 
+* The shiny app should be available in an R package. A template for such a package can be found [here](https://github.com/csoneson/templateDockerShinyPkg).
 
-# goal
+* The docker container is built on top of bioconductor/bioconductor_docker:RELEASE_3_19 and installs the package with the shiny app.
```

Comment:
Is that 3_19 already provided? For now the `Dockerfile` still says 
<img width="466" alt="image" src="https://github.com/hrhotz/templateDockerShiny/assets/7447478/5895f5f3-b20f-4a45-aa4e-9b6efcf85d4f">

Reply:
Yes, it's available - I forgot to change it here, sorry. I used the 3.19 one in https://github.com/csoneson/docker-exploremodelmatrix

Reply:
Perfect, no big deal 😉 
Good to see it up and running already a few days after release!